docs/5-Quick_Reference.adoc

```diff
@@ -1,176 +1,176 @@
-|*{MutableList}* |Analogous to JCF _List_ .
-|*{MutableMap}*  |Analogous to JCF _Map_ .
-|*{MutableSet}*  |Analogous to JCF _Set_ .
-|*{Pair}*        |Container that holds two related objects.
-|===
-
-[cols=",",options="header",]
-[%autowidth]
-|===
-|*Blocks*               |*Description*
-|*{Function}*             |Transforms one type (T) to another (V).
-|*{Function0}*            |A zero-argument lambda (Java 8) that returns some type; a factory.
-|*{Function2}*            |Used by *injectInto* methods; takes the accumulator argument as the first argument, and the current item of the collection as the second argument.
-|*{ObjectIntProcedure}*   |Takes an int as a second argument; this is usually the index of the current element of a collection; doesn't return anything.
-|*{Predicate}*            |Takes an object of some type (T) and returns a boolean.
-|*{Predicate2}*           |Primarily used in methods like *selectWith*, *detectWith*, *rejectWith*. The first argument is the element of the collection being iterated over, and the second argument is a parameter passed into the *Predicate2* from the calling method.
-|*{Procedure}*            |Takes an object of some type (T) and doesn't return anything.
-|*{Procedure2}*           |Takes two objects of types (T1, T2) and doesn't return anything.
-|===
-
-[cols=",",options="header",]
-[%autowidth]
-|===
-|*Static utility Classes* |*Interoperate with*
-|*{ArrayIterate}*         |Arrays
-|*{Iterate}*              |All iterables
-|*{LazyIterate}*          |All iterables for lazy evaluation
-|*{ListIterate}*          |Lists
-|*{MapIterate}*           |Maps
-|*{ParallelArrayIterate}* |Arrays for parallel iteration
-|*{ParallelIterate}*      |Iterables for parallel iteration
-|*{ParallelMapIterate}*   |Maps for parallel iteration
-|*{StringIterate}*        |strings
-|===
-
-
-== Eclipse Collections mindmap
-
-[link=EclipseCollectionsMindMap.png]
-image::EclipseCollectionsMindMap.png[Mindmap,800,945]
-
-[cols="3,^1",]
-[%autowidth]
-|===
-|xref:4-Testing_Utilities.adoc[previous: Unit testing]  |xref:0-RefGuide.adoc[top] 
-|===
+////
+   Copyright (c) 2022 The Bank of New York Mellon.
+   All rights reserved. This program and the accompanying materials
+   are made available under the terms of the Eclipse Public License v1.0
+   and Eclipse Distribution License v. 1.0 which accompany this distribution.
+   The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+   and the Eclipse Distribution License is available at
+   http://www.eclipse.org/org/documents/edl-v10.php.
+////
+== Eclipse Collections Quick Reference
+:sectanchors:
+// Javadoc links
+:api-url:         https://www.eclipse.org/collections/javadoc/11.1.0/org/eclipse/collections
+:rich-url:        https://www.eclipse.org/collections/javadoc/11.1.0/org/eclipse/collections/api/RichIterable.html
+// Interfaces
+:Bag:                 {api-url}/api/bag/Bag.html[Bag]
```

Comment:
There seems to be a formatting issue between the original and current. Not sure if it is spacing or something else.